x-pack/plugins/ml/public/application/model_management/test_models/models/text_expansion/text_expansion_output.tsx

```diff
@@ -0,0 +1,178 @@
+    .slice(0, 5)
+    .map(({ token, value }) => ({ token, value: roundToDecimalPlace(value, 3) }));
+
+  const statInfo = getResultStatFormatting(response);
+
+  return (
+    <>
+      {response.text !== undefined ? (
+        <>
+          <EuiStat
+            title={roundToDecimalPlace(response.score, 3)}
+            textAlign="left"
+            titleColor={statInfo.color}
+            description={
+              <EuiTextColor color={statInfo.color}>
+                <span>
+                  {statInfo.icon !== null ? (
+                    <EuiIcon type={statInfo.icon} color={statInfo.color} />
+                  ) : null}
+                  {statInfo.text}
+                </span>
+              </EuiTextColor>
+            }
+          />
+
+          <EuiSpacer size="s" />
+          <span css={{ color: statInfo.textColor }}>{response.text}</span>
+          <EuiSpacer size="s" />
+        </>
+      ) : null}
+
+      {tokens.length > 0 ? (
+        <EuiAccordion
+          id={`textExpansion_${response.text}`}
+          buttonContent={i18n.translate(
+            'xpack.ml.trainedModels.testModelsFlyout.textExpansion.output.tokens',
+            {
+              defaultMessage: 'Tokens',
+            }
+          )}
+        >
+          <>
+            <EuiSpacer size="s" />
+            <EuiCallOut color="primary">
+              <FormattedMessage
+                id="xpack.ml.trainedModels.testModelsFlyout.textExpansion.output.tokenHelpInfo"
+                defaultMessage="Extracted tokens, which are not synonyms of the query, represent linguistic elements
+              relevant to the search result. The weight value represents the relevancy of a given
+              token."
+              />
+            </EuiCallOut>
+            <EuiSpacer size="s" />
+            <EuiInMemoryTable
+              items={tokens}
+              columns={[
+                {
+                  field: 'token',
+                  name: i18n.translate(
+                    'xpack.ml.trainedModels.testModelsFlyout.textExpansion.output.token',
+                    {
+                      defaultMessage: 'Token',
+                    }
+                  ),
+                },
+                {
+                  field: 'value',
+                  name: i18n.translate(
+                    'xpack.ml.trainedModels.testModelsFlyout.textExpansion.output.weight',
+                    {
+                      defaultMessage: 'Weight',
+                    }
+                  ),
+                },
+              ]}
+            />
+          </>
+        </EuiAccordion>
+      ) : null}
+    </>
+  );
+};
+
+interface ResultStatFormatting {
+  color: string;
+  textColor: string;
+  text: string | null;
+  icon: string | null;
+}
+
+function getResultStatFormatting(response: FormattedTextExpansionResponse): ResultStatFormatting {
+  const { euiColorMediumShade, euiTextSubduedColor, euiTextColor } = euiThemeVars;
```

Comment:
how does it look with the dark theme? I reckon we should not use static values. could you please use [this hook](https://github.com/elastic/kibana/blob/c5eee26671ea5e215e5182b29562f6ffe5476965/x-pack/plugins/ml/public/application/contexts/kibana/use_current_theme.ts#L11) instead?

Reply:
The colours look good in dark mode, but I've switched to the hook anyway.
https://github.com/elastic/kibana/pull/159150/commits/faefba5fff6817c3b79e296a6749e1ed37bed2ef